eng/testing/tests.targets

```diff
@@ -96,51 +96,67 @@
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="$(TestRunErrorMessage)" />
   </Target>
 
- <!-- Generate a self-contained app bundle for Android with tests.
-       This target is executed once build is done for a test lib (after CopyFilesToOutputDirectory target) -->
-  <UsingTask TaskName="AndroidAppBuilderTask" 
-             AssemblyFile="$(ArtifactsObjDir)mono\AndroidAppBuilder\$(TargetArchitecture)\$(Configuration)\AndroidAppBuilder.dll" />
-  <Target Condition="'$(TargetOS)' == 'Android'" Name="BundleTestAndroidApp" AfterTargets="CopyFilesToOutputDirectory">
-    <PropertyGroup>
-      <RuntimePackDir>$(ArtifactsDir)bin\lib-runtime-packs\runtimes\android-$(TargetArchitecture)</RuntimePackDir>
-      <BundleDir>$(OutDir)\Bundle</BundleDir>
-      <AndroidTestRunner>$(RepoRoot)\src\mono\msbuild\AndroidTestRunner\bin</AndroidTestRunner>
-      <AndroidAbi Condition="'$(TargetArchitecture)'=='arm64'">arm64-v8a</AndroidAbi>
-      <AndroidAbi Condition="'$(TargetArchitecture)'=='arm'">armeabi</AndroidAbi>
-      <AndroidAbi Condition="'$(TargetArchitecture)'=='x64'">x86_64</AndroidAbi>
-      <AndroidAbi Condition="'$(AndroidAbi)'==''">$(TargetArchitecture)</AndroidAbi>
-    </PropertyGroup>
-    <!-- TEMP: We need to copy additional stuff into $(OutDir)\Bundle
+  <!-- Common logic for Android/iOS test bundle creation -->
+  <PropertyGroup>
+    <RuntimePackDir Condition="'$(TargetOS)' == 'iOS'">$(ArtifactsDir)bin\lib-runtime-packs\runtimes\ios-$(TargetArchitecture)</RuntimePackDir>
```

Comment:
You should be able to use `<NETCoreAppRuntimePackLibPath>`

Reply:
Yeah I'll need to rebase this anyway after Mitch's PRs go in, I'll make that change then.